support-frontend/assets/helpers/internationalisation/countryGroup.ts

```diff
@@ -29,6 +29,15 @@ export type CountryGroupName =
 	| 'New Zealand'
 	| 'Canada';
 
+export type SupportInternationalisationId =
```

Comment:
```suggestion
export type InternationalisationId =
```

Just a thought - `Support` is not a hugely useful prefix.

Reply:
Thanks James - perhaps one for a future PR